service/controller/clusterapi/v19/resources/namespace/create.go

```diff
@@ -46,7 +46,7 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 
 		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("created namespace %#q in tenant cluster %#q", ns.Name, key.ClusterID(&cr)))
 	} else {
-		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("did not create namespace %#q in tenant cluster %#q", ns.Name, key.ClusterID(&cr)))
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("did not create namespace %#q in tenant cluster %#q", namespace, key.ClusterID(&cr)))
```

Comment:
At this point `ns` is nil so we use the private package constant which is the desired state for logging.